openedx/core/djangoapps/schedules/content_highlights.py

```diff
@@ -14,6 +14,24 @@
 log = logging.getLogger(__name__)
 
 
+def get_all_course_highlights(course_key):
+    """
+    Does the course have any highlights for any section/week in it?
+    This ignores access checks, since highlights may be lurking in currently
+    inaccessible content.
+    Returns a list of all the section highlights in the course
+    """
+    try:
+        course = _get_course_with_highlights(course_key)
+
+    except CourseUpdateDoesNotExist:
+        return []
+
```

Comment:
Nit: remove new lines. This one is very personal style though. Ignore if you want